src/ussd_public.js

```diff
@@ -0,0 +1,501 @@
+go.app = function() {
+    var vumigo = require('vumigo_v02');
+    //var MetricsHelper = require('go-jsbox-metrics-helper'); //..? unique hello mama id..?
+    var App = vumigo.App;
+    var Choice = vumigo.states.Choice;
+    var ChoiceState = vumigo.states.ChoiceState;
+    //var PaginatedChoiceState = vumigo.states.PaginatedChoiceState;
+    var EndState = vumigo.states.EndState;
+    var FreeText = vumigo.states.FreeText;
+
+
+    var GoFC = App.extend(function(self) {
+        App.call(self, 'state_start');
+        var $ = self.$;
+        var interrupt = true;
+
+        self.init = function() {
+
+            // Use the metrics helper to add some metrics
+            //mh = new MetricsHelper(self.im);
+            //mh
+                // Total unique users
+            //    .add.total_unique_users('total.ussd.unique_users')
+
+                // Total sessions
+            //    .add.total_sessions('total.ussd.sessions');
+
+                // Total times reached state_timed_out
+                /*.add.total_state_actions(
+                    {
+                        state: 'state_timed_out',
+                        action: 'enter'
+                    },
+                    'total.reached_state_timed_out'
+                );*/
+
+            // Load self.contact
+            return self.im.contacts
+                .for_user()
+                .then(function(user_contact) {
+                   self.contact = user_contact;
+                });
+        };
+
+
+    // TEXT CONTENT
+
+        var questions = {
+            "state_timed_out":
+                "You have an incomplete registration. Would you like to continue with this registration?",
+            "state_msisdn_permission":  //B
```

Comment:
Tip: rather make the comment `// st-B`, then it's much easier to find related things by searching.